Source/WebCore/style/StyleDifference.cpp

```diff
@@ -65,14 +65,14 @@ static bool positionChangeIsMovementOnly(const Style::InsetBox& a, const Style::
 static bool changeAffectsVisualOverflow(const RenderStyle& a, const RenderStyle& b)
 {
     auto nonInheritedDataChangeAffectsVisualOverflow = [&] {
-        if (&a.nonInheritedData() == &b.nonInheritedData())
+        if (a.m_nonInheritedData.ptr() == b.m_nonInheritedData.ptr())
```

Comment:
Seems like if a data member is intended to be accessed outside the class it’s in, we don’t want to use the `m_` prefix in its name. So I’m surprised to see these changes here. Maybe you have a good style reason for this, related to `friend`?